Dockerfile

```diff
@@ -1,30 +1,39 @@
-# if the image or tag changes, make sure to update the scan structure tool name and version
 FROM soosio/zap2docker-soos as base
```

Comment:
// TODO still before merging
Finish structuring everything, move classes to files, etc.
Make logging close as possible to python one.
Finish missing functionalities, like SARIF.
Decide on missing parameters/capabilities left out on purpose.
Setup test
Modify github actions
Add helpformated.
Modify readme.md